test/Ocelot.AcceptanceTests/Steps.cs

```diff
@@ -476,6 +477,33 @@ public void GivenOcelotIsRunningWithMiddlewareBeforePipeline<T>(Func<object, Tas
         _ocelotClient = _ocelotServer.CreateClient();
     }
 
+    public void GivenOcelotIsRunningWithSpecificHandlerRegisteredInDi<T>()
```

Comment:
I guess this helper has 1 reference only.
Why not to define in the testing class?
Also another helpers in `Steps` should help to write less code. DRY?

Reply:
Because, it was easier to implement it like that

Reply:
OK, we will have more helpers with one reference. Great!

Reply:
Duplicate of the method on the line 567 with the following signature:
```c#
public void GivenOcelotIsRunningWithGlobalHandlerRegisteredInDi<TOne>()
        where TOne : DelegatingHandler
```

Reply:
Ok, this one makes sense @raman-m